File: Ejercicios en clases/poo java/Two java/library/Rectangle.java
public class Rectangle extends GeometricFigure {
    private double base;
    private double height;

    public Rectangle(double base, double height) {
        super("Rectangle");
        this.base = base;
        this.height = height;
    }

    public double getBase() {
        return base;
    }

    public double getHeight() {
        return height;
    }

    @Override
    public double calculateArea() {
        return base * height;
    }
}

File: Ejercicios en clases/poo php/Two php/Library/rectangle.php
<?php

class rectangle extends geometricFigures {
    private float $base;
    private float $height;
    private float $calculate;

    public function __construct(float $base, float $height) {
        parent::__construct("Rectangle");
        $this->base=$base;
        $this->height = $height;
    }

    public function getBase(): float {
        return $this->base;
    }

    public function getHeight(): float {
        return $this->height;
    }

    public function calculateArea(): float {
        $this->calculate = $this->getbase() * $this->getHeight();
        return $this->calculate;
    }
}
